Makefile

```diff
@@ -75,7 +75,7 @@ $(NODE_EXE): config.gypi out/Makefile
 
 $(NODE_G_EXE): config.gypi out/Makefile
 	$(MAKE) -C out BUILDTYPE=Debug V=$(V)
-	if [ ! -r $(NODE_EXE) -o ! -L $(node_EXE) ]; then ln -fs out/Debug/$(NODE_EXE) $@; fi
+	if [ ! -r $(NODE_EXE) -o ! -L $(NODE_EXE) ]; then ln -fs out/Debug/$(NODE_EXE) $@; fi
```

Comment:
Should the `$(NODE_EXE)` be `$(NODE_G_EXE)` for consistency? Or in fact could all of the `$(NODE_EXE)` in this line be `$@`?